templates/vhost/_directories.erb

```diff
@@ -37,7 +37,7 @@
     <%- if ! directory['geoip_enable'].nil? -%>
     GeoIPEnable <%= scope.call_function('apache::bool2httpd', [directory['geoip_enable']]) %>
     <%- end -%>
-    <%- if directory['options'] -%>
+    <%- if Array(directory['options']).length > 0 -%>
```

Comment:
I think you need to check if the option is set at all to be safe. Then it's also better to use `any?`
```suggestion
    <%- if directory['options'] && Array(directory['options']).any? -%>
```